crates/accelerate/src/unitary_synthesis.rs

```diff
@@ -460,18 +516,13 @@ fn run_2q_unitary_synthesis(
                                     inst_qubits,
                                 ),
                                 None => (
-                                    sequence
-                                        .decomp_gate
-                                        .operation
-                                        .standard_gate()
-                                        .name()
-                                        .to_string(),
+                                    sequence.decomp_gate.name().to_string(),
                                     Some(params.iter().map(|p| Param::Float(*p)).collect()),
                                     inst_qubits,
                                 ),
                             }
                         });
-                let synth_error_from_target = synth_error(py, scoring_info, target);
+                let synth_error_from_target = synth_error(py, scoring_info, target.unwrap());
```

Comment:
Why `target` can't be `None` here?

Reply:
Good question. If `target` is `None`, we would be in the `basis_gates` path, which proposes a single decomposer, so we would avoid computing the synthesis score (https://github.com/Qiskit/qiskit/pull/13704/files#diff-c37693939c4bc977d8d0d823e462802e87616250233241a010553702515f4da3R494). Maybe I can try to make the behavior more explicit...